cdap-tms/src/main/java/co/cask/cdap/messaging/service/CoreMessagingService.java

```diff
@@ -247,6 +267,29 @@ protected void startUp() throws Exception {
     LOG.info("Core Messaging Service started");
   }
 
+  /**
+   * Creates the given topic if it is not yet created. Adds a topic to the creationFailureTopics if creation fails.
+   */
+  private void createSystemTopic(String topicName,  Queue<TopicId> creationFailureTopics) {
+    TopicId topicId;
+    try {
```

Comment:
Either just log the `topicName` instead of do:

```java
try {
  TopicId topicId = NamespaceId.SYSTEM.topic(topicName);
  try {
    createTopicIfNotExists(topicId);
  } catch (Exception e) {
    LOG.warn("Topic {} creation failed with exception {}. Will retry.", topicId, e.getMessage());
    LOG.debug("Topic {} creation failure stacktrace", topicId, e);
    creationFailureTopics.add(topicId);
  }
} catch (IllegalArgumentException e) {
  LOG.warn(String.format("Ignore creation of invalid topic '%s'", topicName), e);
}